kotest-assertions/kotest-assertions-shared/src/commonMain/kotlin/io/kotest/assertions/NondeterministicHelpers.kt

```diff
@@ -2,5 +2,6 @@ package io.kotest.assertions
 
 typealias SuspendingProducer<T> = suspend () -> T
 
-
+/** @return empty string when count is one, otherwise a single 's'. */
+fun plural_s(count: Int) = if(count == 1) "" else "s"
```

Comment:
not sure about this naming convention, looks kinda weird to me

Reply:
yeah, I don't usually name functions like that, but it seemed the most appropriate here - `pluralS` or `pluralize` or whatever don't seem much better